### ISSUE TITLE
Questions about codes

### ISSUE THREAD

**Issue by Blcony** [gengshan-y/VCN#2]
Hi gengshan~

Thanks for your wonderful work and code!

When I tried to train the VCNet according to the "Pretrain on flying chairs and flying things" part of readme, I found an error which is "**one of the variables needed for gradient computation has been modified by an inplace operation**". But I just modified some paths for my own environment, and didn't change other parts. 

I am so confused about this error, so I wonder that: (1) have you met this error before? (2) can you give me some suggestions about solving this problem? 

Thanks for your help! And I wish to hear from you soon!

Best wishes.

Blcony

**Comment by gengshan-y** [comment]
I didn't see the issue using pytorch 1.2.0 for training. What's the pytorch version are you using? I'll try to reproduce this error.

**Comment by Blcony** [comment]
I used the pytorch 1.0.1 for training, and the problems haven't been solved now.

**Comment by gengshan-y** [comment]
Not sure what happened. One simple fix though is to upgrade pytorch to 1.1.0, for example
`conda install pytorch==1.1.0 torchvision==0.3.0 cudatoolkit=9.0 -c pytorch
`
